athena-neptune/docs/aws-glue-sample-scripts/PropertyGraph.md

```diff
@@ -1,141 +1,64 @@
-
-If you're planning to use your own data set instead of the Air Routes sample dataset, then you need to modify the script according to your data structure. 
-
-Ensure to have the right executable permissions on the script once you download it.
-
-```
-chmod 755 sample-cli-script.sh
-```
-Ensure to setup credentials for your AWS CLI to work.
-
-Replace &lt;aws-profile> with the AWS profile name that carries your credentials and replace &lt;aws-region> with AWS region where you are creating the AWS Glue tables which should be the same as your Neptune Cluster's AWS region.
-
-```
-./sample-cli-script.sh  <aws-profile> <aws-region>
-```
-
-
-If all goes well you now have the Glue Database and Tables that are required for your Athena Neptune Connector setup and you can move on to those steps mentioned [here](../neptune-connector-setup/).
-
-### Sample table post setup
+The next screenshot shows columns and advanced properties for the sample `airport` table that maps to airport vertices in Neptune. It is a vertex table, indicated the `componenttype` of `vertex`. Its columns include `id` (the airport vertex ID) plus `type`, `code`, `icao`, and `desc` (vertex properties).
 
 ![](./assets/table.png)
 
-### Query examples
-
-##### Graph Query
+Here is an edge table for `route`. Columns include built-in `id`, `out`, and `in`. The `dist` column maps to an edge property of the `route` edge.
 
-```
-g.V().hasLabel("airport").as("source").out("route").as("destination").select("source","destination").by(id()).limit(10)
-```
-
-#####  Equivalent Athena Query
-```
-SELECT 
-a.id as "source",b.id as "destination" FROM "graph-database"."airport" as a 
-inner join "graph-database"."route" as b 
-on a.id = b.out
-inner join "graph-database"."airport" as c 
-on c.id = b."in"
-limit 10;
-```
+![](./assets/table_route.png)
 
-## Custom query
+Finally, here is a table that presents a custom view. Notice `componentype` is `view`. 
 
-Neptune connector custom query feature allows you to specify a custom Glue table, which matches response of a Gremlin Query. For example a gremlin query like 
+![](./assets/table_custom.png)
 
+The `query` property is 
 ```
-g.V().hasLabel("airport").as("source").out("route").as("destination").select("source","destination").by(id()).limit(10)
-
+g.V().hasLabel("airport").as("source").out("route").as("destination").select("source","destination").by("code").limit(10)
 ```
 
-matches to a Glue table 
-
-![](./assets/customquery-exampletable.png)
-
-Refer example scripts on how to create a table [here](./manual/sample-cli-script.sh)
+Columns are `source` and `destination`, which are the values returned by the Gremlin query above.
 
-> **NOTE**
->
-> Custom query feature allows simple type (example int,long,string,dateime) projections as query output
+Run SQL queries against the Athena service to retrieve this property graph data. 
 
-
-### Example query patterns 
```

Comment:
why are we removing all these examples?

Reply:
The documentation has been restructured. but all important detail is still there in the docs. We did decide, to make the example easier to follow, to use CloudFormation rather than CLI and SDK to deploy the Glue catalog.